client/src/pages/Home.js

```diff
@@ -1,77 +1,49 @@
-import { useEffect, useState } from "react";
+import { useState } from "react";
 import { Link } from "react-router-dom";
 
 import "./Home.css";
 import ModalVideo from "../components/ModalVideo";
 import map from "./img/map.jpg";
 import CityIcon from "./CityIcon";
 
+import rhythmsJsonData from "./rhythmsData.json";
+
 export function Home() {
-	const [message, setMessage] = useState("Loading...");
+
 	const [show, setShow] = useState(false);
 	const [selectedIcon, setSelectedIcon] = useState(null);
-    const handleClose = () => setShow(false);
-    const handleShow = (videoInfo) => {
+
+	const handleClose = () => setShow(false);
+	const handleShow = (videoInfo) => {
 		setShow(true);
 		setSelectedIcon(videoInfo); // Use a state variable to store the selected video info
 	};
 
-	useEffect(() => {
-		fetch("/api")
-			.then((res) => {
-				if (!res.ok) {
-					throw new Error(res.statusText);
-				}
-				return res.json();
-			})
-			.then((body) => {
-				setMessage(body.message);
-			})
-			.catch((err) => {
-				console.error(err);
-			});
-	}, []);
```

Comment:
I'd rather the data come from this useEffect than from a JSON file. If you pull that same data from the API, I'll approve